package.json

```diff
@@ -20,12 +21,13 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5.0.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^2.1.1",
     "text-encoding": "^0.7.0"
   },
   "devDependencies": {
+    "@semantic-ui-react/css-patch": "^1.0.0",
```

Comment:
This workaround is from https://github.com/Semantic-Org/Semantic-UI-React/issues/4287#issuecomment-1016579332